.gitmodules

```diff
@@ -54,7 +54,7 @@
 	url = https://github.com/rpm-software-management/rpm
 [submodule "libraries/cmake/source/popt/src"]
 	path = libraries/cmake/source/popt/src
-	url = https://github.com/osquery/third-party-popt
+	url = https://github.com/rpm-software-management/popt.git
```

Comment:
Should we also archive the `osquery/third-party-popt` repo?

Reply:
> Should we also archive the `osquery/third-party-popt` repo?

Totally!